File: der/der.go
package main

import "fmt"

func calculateDER(putouts, assists, totalChances float64) float64 {
    // DER = (putouts + assists) / totalChances
    return (putouts + assists) / totalChances
}

func main() {
    putouts := 120.0
    assists := 50.0
    totalChances := 200.0

    DER := calculateDER(putouts, assists, totalChances)
    fmt.Printf("Defensive Efficiency Rating (DER): %.3f\n", DER)
}

File: der/der.py
def calculate_DER(putouts, assists, total_chances):
    """
    Calculate Defensive Efficiency Rating (DER) for a first baseman.
    
    DER = (putouts + assists) / total_chances
    
    Args:
    putouts (int): Number of putouts by the first baseman.
    assists (int): Number of assists by the first baseman.
    total_chances (int): Total defensive chances (putouts + assists + errors) for the first baseman.
    
    Returns:
    float: Defensive Efficiency Rating (DER).
    """
    return (putouts + assists) / total_chances
